test/models/upvote_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class UpvoteTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+	test "Comment ID uniqueness" do 
+		upvote = Upvote.new(:user_id => 123123 , :comment_id => 456456)
```

Comment:
Instead of using the ID directly here you can get rails to do this for you!

You have to pull up the user
```
user = users(:bob) #the bob key comes from line 3 of users.yml
```

and then do 
```
Upvote.new(:user => user, comment_id => 456456)
```

You can do the same for comment if you change `upvote.yml` so that the `comment_id` points to a valid `comment`